README.md

```diff
@@ -1,75 +1,36 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Protofolio
 
-> One paragraph statement about the project.
+> I have tried to design a mobile version of protofilio.
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
-
-## Built With
-
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+It is a Protofolio template mobile version and I am working on it to make a great responsive protofolio template.
```

Comment:
- [ ] [SUGGESTION] Improve your README file, try to explain what you did, what kind of technologies or techniques you used, highlight the best of your project, what is the main scope, and so on. 😉